app/views/doorkeeper/authorizations/new.html.erb

```diff
@@ -19,22 +19,22 @@
     </div>
   <% end %>
 
-  <div class="actions">
-    <%= form_tag oauth_authorization_path, :method => :post, :style => 'display: inline-block' do %>
+  <div>
+    <%= form_tag(oauth_authorization_path, :method => :post, 'data-abide' => '') do %>
```

Comment:
If you add class `button_to` to each of these `form_tag`s, it will make them display inline and not stack up on top of each other.